app/command/put_publish_intent.rb

```diff
@@ -0,0 +1,17 @@
+class Command::PutPublishIntent < Command::BaseCommand
+  attr_reader :event
+
+  def call
+    url_arbiter.reserve_path(
+      base_path,
+      publishing_app: content_item[:publishing_app]
+    )
+
+    live_content_store.put_publish_intent(
+      base_path: base_path,
+      publish_intent: content_item
+    )
+  rescue GOVUK::Client::Errors::HTTPError => e
```

Comment:
It's a little unclear that this specific type of HTTP error can only (currently) come from the URL arbiter client, but I'm not sure how we'd make that clearer right now.